pepys_import/file/importer.py

```diff
@@ -52,6 +52,21 @@ def can_load_this_file(self, file_contents) -> bool:
         :rtype: bool
         """
 
+    def init_load_this_file(self, data_store, path, file_object, datafile):
+        """Initialises the loading of this data file
+
+        Performs the common operations that must be performed before the
+        load_this_file method is called. This *must* be called immediately
```

Comment:
> This *must* be called immediately

We can avoid this issue by calling the "child" method ourselves, from the end of this "parent" one.

Let me try it out.